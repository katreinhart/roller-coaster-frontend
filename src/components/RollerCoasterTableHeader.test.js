import React from 'react'
import { shallow } from 'enzyme'

import RollerCoasterTableHeader from './RollerCoasterTableHeader'

describe('RollerCoasterTableHeader', () => {
  const rcth = shallow(<RollerCoasterTableHeader />)

  it('renders without crashing', () => {
    expect(rcth).toMatchSnapshot()
  })
})