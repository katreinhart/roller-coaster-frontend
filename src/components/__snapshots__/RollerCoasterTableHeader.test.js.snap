// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RollerCoasterTableHeader renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RollerCoasterTableHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <tr>
        <th>
                #
        </th>
        <th>
                Name
        </th>
        <th>
                Park
        </th>
        <th>
                City
        </th>
        <th>
                State
        </th>
        <th
                className="buttonFixedSize"
        >
                Edit
        </th>
        <th
                className="buttonFixedSize"
        >
                Delete
        </th>
</tr>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <th>
            #
</th>,
          <th>
            Name
</th>,
          <th>
            Park
</th>,
          <th>
            City
</th>,
          <th>
            State
</th>,
          <th
            className="buttonFixedSize"
>
            Edit
</th>,
          <th
            className="buttonFixedSize"
>
            Delete
</th>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "#",
          },
          "ref": null,
          "rendered": "#",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Name",
          },
          "ref": null,
          "rendered": "Name",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Park",
          },
          "ref": null,
          "rendered": "Park",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "City",
          },
          "ref": null,
          "rendered": "City",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "State",
          },
          "ref": null,
          "rendered": "State",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Edit",
            "className": "buttonFixedSize",
          },
          "ref": null,
          "rendered": "Edit",
          "type": "th",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Delete",
            "className": "buttonFixedSize",
          },
          "ref": null,
          "rendered": "Delete",
          "type": "th",
        },
      ],
      "type": "tr",
    },
    "type": "thead",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <tr>
          <th>
                    #
          </th>
          <th>
                    Name
          </th>
          <th>
                    Park
          </th>
          <th>
                    City
          </th>
          <th>
                    State
          </th>
          <th
                    className="buttonFixedSize"
          >
                    Edit
          </th>
          <th
                    className="buttonFixedSize"
          >
                    Delete
          </th>
</tr>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <th>
              #
</th>,
            <th>
              Name
</th>,
            <th>
              Park
</th>,
            <th>
              City
</th>,
            <th>
              State
</th>,
            <th
              className="buttonFixedSize"
>
              Edit
</th>,
            <th
              className="buttonFixedSize"
>
              Delete
</th>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "#",
            },
            "ref": null,
            "rendered": "#",
            "type": "th",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Name",
            },
            "ref": null,
            "rendered": "Name",
            "type": "th",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Park",
            },
            "ref": null,
            "rendered": "Park",
            "type": "th",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "City",
            },
            "ref": null,
            "rendered": "City",
            "type": "th",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "State",
            },
            "ref": null,
            "rendered": "State",
            "type": "th",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Edit",
              "className": "buttonFixedSize",
            },
            "ref": null,
            "rendered": "Edit",
            "type": "th",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Delete",
              "className": "buttonFixedSize",
            },
            "ref": null,
            "rendered": "Delete",
            "type": "th",
          },
        ],
        "type": "tr",
      },
      "type": "thead",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
